frontend/resources/styles/common/base.scss

```diff
@@ -18,6 +18,11 @@ $lh-115: 1.15;
 $lh-133: 1.33;
 $size-4: 1rem;
 
+/* Background for storybook */
+body.sb-show-main {
+  background-color: var(--color-background-secondary);
+}
+
```

Comment:
Not the right way to do this... If we want to set background colors for stories, we have https://storybook.js.org/docs/essentials/backgrounds 

If it's the background color of storybook itself, there's themes https://storybook.js.org/docs/configure/user-interface/theming

Reply:
No, it won't work because in the global configuration `preview.js` we have:

```
    },
    backgrounds: { disable: true },
``` 
This will pick the global styles. And this is why y added the custom here.